web/src/beta/features/PluginPlayground/Viewer/hooks.ts

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useMemo, useRef, useState } from "react";
+
+import { Camera } from "@reearth/beta/utils/value";
+import { MapRef } from "@reearth/core";
+import { config } from "@reearth/services/config";
+
+export default () => {
+  // Refrence: hooks of Editor/Visualizer/hooks.ts and Publish/hooks.ts
+  const visualizerRef = useRef<MapRef | null>(null);
+  const [ready, setReady] = useState(false);
+  const [currentCamera, setCurrentCamera] = useState<Camera | undefined>(undefined);
+
+  const engineMeta = useMemo(
+    () => ({
+      cesiumIonAccessToken: config()?.cesiumIonAccessToken,
+    }),
+    [],
+  );
+
+  const sceneProperty = useMemo(
+    () => ({
+      tiles: [
+        {
+          id: "default",
+          type: "default",
+        },
+      ],
+    }),
+    [],
+  );
+
+  useEffect(() => {
+    setReady(true);
+  }, []);
```

Comment:
Sorry, I am trying to understand why we `setReady` to `true` here?

Reply:
`ready` is passing to core Visualizer and has a long history i guess. To my understanding it can be used to make sure data is ready before engine really renders. I left this logic here just in case in the future you need to make sure some data is there, but for now i think remove it and directly passing a ready=true also works.

Reply:
I'll mark this as resolved since this part is just a temp implementation.